modules/social_features/social_user/social_user.install

```diff
@@ -208,3 +208,21 @@ function social_user_update_11402() : void {
   ]);
   $form_mode->save();
 }
+
+/**
+ * Update 'views.view.user_admin_people'.
+ *
+ * Remove last access from display, and add last login field to display.
+ */
+function social_user_update_11403(array &$sandbox): void {
+  $config_file = \Drupal::service('extension.list.module')->getPath('social_user') . '/config/static/social_user_update_11403.yml';
+
+  if (is_file($config_file)) {
+    $settings = Yaml::parse((string) file_get_contents($config_file));
+    if (is_array($settings)) {
+      $config = \Drupal::configFactory()
+        ->getEditable('views.view.user_admin_people');
+      $config->setData($settings)->save(TRUE);
```

Comment:
@Kovalskiy266 Is it really needed to re-write whole config file here?
Can we just update fields that needed using `update_helper` module?
As we did for example in`social_user_update_11401`

Reply:
@Taras Kruts Hello!
I tried to do it before with update_helper, but when I removed the last access field and added the last login field, the last login field was added to the end, so it followed the operations field, which is not good.
![image](https://github.com/goalgorilla/open_social/assets/85495223/a5f06225-2dd9-4eb4-9b18-060a4bb2ff81)

 I tried adding a weight to the field, but it didn't work. Is there any way to place the last login field instead of last access using update_helper?